bindings/protocols/coap/index.html

```diff
@@ -61,79 +61,130 @@ <h2>Protocol Overview</h2>
         </p>
         <!-- Maybe RFC numbers could be included in the table? -->
-        <table class="def">
-            <thead>
-                <tr>
-                    <th>Feature</th>
-                    <th>Covered by</th>
-                </tr>
-            </thead>
-            <tbody>
-                <tr>
-                    <td>Request to Server</td>
-                    <td><code>"href": ..., "coap:method": ...</code></td>
-                </tr>
-                <tr>
-                    <td>Request to Forward Proxy</td>
-                    <td><code>"href": ..., "coap:method": ..., "coap:proxy": ...</code></td>
-                </tr>
-                <tr>
-                    <td>Caching: Validation</td>
-                    <td>n/a</td>
-                </tr>
-                <tr>
-                    <td>Conditional Requests</td>
-                    <td>n/a</td>
-                </tr>
-                <tr>
-                    <td>Content</td>
-                    <td><code>"coap:format": ...</code></td>
-                </tr>
-                <tr>
-                    <td>Content Negotiation</td>
-                    <td><code>"coap:accept": ...</code></td>
-                </tr>
-                <tr>
-                    <td>Observing Resources</td>
-                    <td><code>"subprotocol": "coap:observe"</code></td>
-                </tr>
-                <tr>
-                    <td>Block-Wise Transfer</td>
-                    <td><code>"coap:blockwise": { ... }</code></td>
-                </tr>
-                <tr>
-                    <td>Quick Block-Wise Transfer</td>
-                    <td><code>"coap:qblockwise": { ... }</code></td>
-                </tr>
-                <tr>
-                    <td>Hop Limit</td>
-                    <td><code>"coap:hoplimit": ...</code></td>
-                </tr>
-                <!--
-                <tr>
-                    <td>OSCORE</td>
-                    <td><code>SecurityScheme</code></td>
-                </tr>
-                <tr>
-                    <td>EDHOC</td>
-                    <td><code>SecurityScheme</code></td>
-                </tr>
-                -->
-            </tbody>
-        </table>
+        <section>
+            <h3>Requests</h3>
+            <p>
+                ... <code>href</code> and <code>coap:method</code>.
+            </p>
+            <pre id="example-feature-request" class="example" title="Requests">
+{
+    "op": [ "readproperty" ],
+    "href": "coap://[2001:DB8::1]/.../...",
+    "coap:method": "GET"
+}
+</pre>
+            <p>
+                ... <code>coap:format</code>.
+            </p>
+            <pre id="example-feature-format" class="example" title="Payload Format">
+{
+    "op": [ "writeproperty" ],
+    "href": "coap://[2001:DB8::1]/.../...",
+    "coap:method": "PUT",
+    "coap:format": 112  // application/senml+cbor
```

Comment:
I just remembered that the default `contentType` in forms is actually `application/json`. So maybe this should rather look like this? (Even though it causes redundancy.)


```suggestion
    "coap:format": 112,
    "contentType": "application/senml+cbor"
```

Reply:
Yes this is definitely needed

Reply:
Good point. However, this raises the question how `contentType` is supposed to work for protocols that don't use the concept of Content Types. CoAP's Content Format is close enough, but what about other protocols where, for example, the protocol defines the wire format and there are not matching media types? It seems to me that `contentType` might actually be something HTTP specific and shouldn't have a default for other protocols...

Reply:
Let's continue this discussion in #159.